bitnami/etcd/templates/cronjob.yaml

```diff
@@ -57,7 +69,13 @@ spec:
               subPath: save-snapshot.sh
             - name: snapshot-volume
               mountPath: /snapshots
+            - name: certs
+              mountPath: /opt/bitnami/etcd/certs/client
+              readOnly: true
           volumes:
+          - name: certs
+            secret:
+              secretName: etcd-client-certs
```

Comment:
These `volumes` and `volumeMounts` should only be added to the cronjob manifest when `auth.client.secureTransport=true`, right?

Reply:
The `secretName` should not be hardcoded to any value. Please check how it's implemented in the etcd sts:

```yaml
          secretName: {{ required "A secret containinig the client certificates is required" .Values.auth.client.existingSecret }}
          defaultMode: 256
```

Reply:
Yes, so i'll add another if statement there on the `volume` and `volumeMounts` and i'll fix the `secretName`